holmes/plugins/toolsets/bash/parse_command.py

```diff
@@ -1,39 +1,108 @@
-from holmes.plugins.toolsets.bash.grep import create_grep_parser, stringify_grep_command
-from holmes.plugins.toolsets.bash.kubectl import (
-    create_kubectl_parser,
-    stringify_kubectl_command,
-)
+from holmes.plugins.toolsets.bash.kubectl import KubectlCommand
+from holmes.plugins.toolsets.bash.aws import AWSCommand
+from holmes.plugins.toolsets.bash.azure import AzureCommand
+from holmes.plugins.toolsets.bash.argocd import ArgocdCommand
+from holmes.plugins.toolsets.bash.docker import DockerCommand
+from holmes.plugins.toolsets.bash.helm import HelmCommand
+
+# Utilities imports - all now use Command classes
+from holmes.plugins.toolsets.bash.utilities.wc import WCCommand
+from holmes.plugins.toolsets.bash.utilities.cut import CutCommand
+from holmes.plugins.toolsets.bash.utilities.sort import SortCommand
+from holmes.plugins.toolsets.bash.utilities.uniq import UniqCommand
+from holmes.plugins.toolsets.bash.utilities.head import HeadCommand
+from holmes.plugins.toolsets.bash.utilities.tail import TailCommand
+from holmes.plugins.toolsets.bash.utilities.tr import TrCommand
+from holmes.plugins.toolsets.bash.utilities.base64_util import Base64Command
+from holmes.plugins.toolsets.bash.utilities.jq import JqCommand
+from holmes.plugins.toolsets.bash.utilities.sed import SedCommand
+from holmes.plugins.toolsets.bash.utilities.grep import GrepCommand
+
+
+# All commands now use BashCommand classes
+commands: list[BashCommand] = [
+    WCCommand(),
+    KubectlCommand(),
+    AWSCommand(),
+    AzureCommand(),
+    ArgocdCommand(),
+    DockerCommand(),
+    HelmCommand(),
+    GrepCommand(),
+    CutCommand(),
+    SortCommand(),
+    UniqCommand(),
+    HeadCommand(),
+    TailCommand(),
+    TrCommand(),
+    Base64Command(),
+    JqCommand(),
+    SedCommand(),
+]
+
+
+class QuietArgumentParser(argparse.ArgumentParser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _print_message(self, message, file=None):
+        if message:
+            logging.debug(message.strip())
+
+    def error(self, message):
+        logging.debug(f"Error: {message}")
+        self.exit(2)
 
 
 def create_parser() -> argparse.ArgumentParser:
-    parser = argparse.ArgumentParser(
-        description="Parser for commands", exit_on_error=False
+    parser = QuietArgumentParser(
+        prog="command_parser",  # Set explicit program name
+        description="Parser for commands",
+        exit_on_error=False,
+        add_help=False,  # Disable help to avoid conflicts with -h in subcommands
     )
     commands_parser = parser.add_subparsers(
         dest="cmd", required=True, help="The tool to command (e.g., kubectl)"
     )
 
-    create_kubectl_parser(commands_parser)
-    create_grep_parser(commands_parser)
+    # Add all BashCommand classes
+    for command in commands:
+        command.add_parser(commands_parser)
+
     return parser
 
 
+def validate_command(
```

Comment:
Is that intentional? Not sure i understand what is going on here (I do see that it is used)